tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.6
+
+LABEL maintainer "RPS <devops@rpsgroup.com>"
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir /glider-dac-status/logs
+COPY /web/ /glider-dac-status/web
+COPY app.py config.yml flask_environments.py manage.py /glider-dac-status/
+COPY status /glider-dac-status/status
+COPY navo /glider-dac-status/navo
+COPY requirements/requirements.txt /requirements.txt
+COPY tests/test_status.py /glider-dac-status/
+RUN rm -f /glider-dac-status/test/test_status.py
+COPY requirements/test_requirements.txt /test_requirements.txt
+
+WORKDIR /glider-dac-status
+
+RUN apt-get update && \
+    apt-get install -y python3-netcdf4 && \
+    pip install --no-cache cython gunicorn && \
+    pip install --no-cache -r /requirements.txt && \
+    pip install --no-cache -r /test_requirements.txt && \
+    rm -rf /var/lib/apt/lists/*
+
```

Comment:
It's good practice to become a non-root user if you don't need to be running tasks as root in the container steps.  Once you've created the account (which could be appended to the above instruction with `&& useradd <USERNAME>`), you can use the Dockerfile `USER <USERNAME>`.  You may have to adjust directory permissions if tests need to write files in certain directories.